src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/SlimefunGuideListener.java

```diff
@@ -72,16 +74,24 @@ else if (openGuide(e, SlimefunGuideLayout.CHEAT_SHEET) == Result.ALLOW) {
             }
         }
     }
+    
+    @ParametersAreNonnullByDefault
+    private void openGuide(Player p, PlayerRightClickEvent e, SlimefunGuideLayout layout) {
+        GuideOpenEvent event = new GuideOpenEvent(p, e.getItem(), layout) ;
```

Comment:
Haha, build is failing.
```suggestion
        SlimefunGuideOpenEvent event = new SlimefunGuideOpenEvent(p, e.getItem(), layout);
```